src/main/java/cms/gov/madie/measure/repositories/MeasureSearchServiceImpl.java

```diff
@@ -66,6 +66,11 @@ && isNumeric(versionParts[1])
                 && isNumeric(versionParts[2])) {
               Criteria otherCriteria = Criteria.where("version").is(Version.parse(searchField));
               orConditions.add(otherCriteria);
+            } else {
```

Comment:
In these cases you want to do an else-if rather than the nested else then if, since there is only 1 option. Ultimately though it does pretty much the same thing.